pp/integrations/plugin-packs/procedures/applications-webservers-iis-restapi.md

```diff
@@ -82,6 +82,8 @@ Here is the list of services for this connector, detailing all metrics linked to
 
 To use this Monitoring Connector, you must enable the Microsoft IIS Administration API. Microsoft provides an official documentation to achieve this: https://docs.microsoft.com/en-us/iis-administration/
 
+> Warning : IIS Administration API component must be at least in version 2.3.0 (otherwise errors may be returned if the supervised server is not in UTC timezone).
```

Comment:
```suggestion
> Warning : The IIS Administration API component must be at least in version 2.3.0 (otherwise errors may be returned if the monitored server is not in the UTC timezone).
```